maltoolbox/language/languagegraph.py

```diff
@@ -97,12 +97,60 @@
     },
     'Disabled':
     {
-        'arguments': [1.0],
+        'arguments': [0.0],
         'name': 'Bernoulli',
         'type': 'function'
     },
 }
 
+def get_ttc_distribution(
+        step_dict: dict,
+        defense_default_ttc = None,
+        attack_default_ttc = None
+    ) -> Optional[dict]:
+    """
+    If the TTC provided is a predefined name replace it with the
+    probability distribution it corresponds to. Otherwise, simply return
+    the TTC distribution provided as is.
+
+    Arguments:
+    ttc_entry   - the TTC entry to check for predefined names
+
+    Returns:
+    If the TTC entry provided contained a predefined name the TTC
+    probability distrubtion corresponding to it. Otherwise, the TTC
+    distribution provided as a parameter as is.
+    """
+
+    # step_ttc can be either: None, predefined distribution or explicit
+    # distribution. The goal is to return explicit distribution for all steps
```

Comment:
I don't like the phrasing here for multiple reasons in that last sentence.

Could we change it to something like "This method will convert predefined TTC names and apply defaults to ensure uniformity(explicit distributions)."